JavaScript/README.md

```diff
@@ -2,107 +2,117 @@
 
 ## Getting Started
 
-Recognizer's are organized into groups and designed to be used in C# and Node.js to help you build great applications! To use the samples clone our GitHub repository using Git.
+Recognizer's are organized into groups and designed to be used in C# and Node.js to help you build great applications! To use the samples, install the `recognizers-text` package, or  clone our GitHub repository using Git.
+
+## Cloning and building the Repository
 
     git clone https://github.com/southworkscom/Recognizers-Text.git
```

Comment:
This address should use Microsoft's repo.

Reply:
Good catch! Fixed